joomla/joomla-cms
# [4.4][bugfix] Use iife for the scripts core and validate

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/43714 .

### Summary of Changes

Use iife instead of esm for the scripts core and validate


### Testing Instructions

- Apply the PR
- run `npm install`
- Check that authenticating in the administrator works
- Create a new article, don't add anything and press save, the validation error should appear
- still in the article edit view, open the browser console, and type `const initialize = 1;` and hit Enter.

Before: `Error SyntaxError: Identifier 'initialize' has already been declared`
After: No error

### Actual result BEFORE applying this Pull Request

Leaking variables in the global scope

### Expected result AFTER applying this Pull Request

NO leaking variables in the global scope

### Link to documentations
Please select:
- [] Documentation link for docs.joomla.org: <link>


@Fedik
